src/web/app/components/student-list/student-list.component.scss

```diff
@@ -2,6 +2,15 @@
   text-align: center;
 }
 
+@media (max-width: 768px) { //applies only to mobile screens
+  .custom-button {
+    font-size: 14px;
```

Comment:
Do you think we could remove this `font-size`? I toggled it on and off and couldn't tell the difference; prefer keeping to the `0.875rem` defined since there's some [accessibility-related concerns](https://austingil.com/px-or-rem-in-css/).